### ISSUE TITLE
Lets rename account to space

### ISSUE THREAD

**Issue by Gozala** [web3-storage/w3-protocol#142]
In the [internal discussion](https://filecoinproject.slack.com/archives/C02BZPRS9HP/p1667853995854219) @jchris proposed that we:

> based on this and the previous note, I think we should rename `accounts` to `spaces` in the code ASAP. Otherwise spaces and whatever the real account domain object turns out to be are gonna get muddied.

Here some more reasoning from @jchris on why from that thread

> the problem is that the thing we are calling accounts is not accounts, we have something else we will call accounts. Space are things each account may have many of. (I'm assuming an account is the thing tied to a billing contact, and that you might want to fund more than one space from the same account.)

**Comment by jchris** [comment]
I'm guessing that an account will be a business object of web3.storage, and may not even be represented in the protocol.

**Comment by gobengo** [comment]
@Gozala in your model is there a meaningful difference between what you call 'memory space' and what you imagine I might mean by 'storage space'? Since we use 'storage' so many places already (eg web3.storage), I'm wondering if you introduced the 'memory' terminology intentionally to distinguish from 'storage'.

**Comment by Gozala** [comment]
> @Gozala in your model is there a meaningful difference between what you call 'memory space' and what you imagine I might mean by 'storage space'? 

No. I also tend to write (memory) space and  think if it as qualifier not really as a separate concept

Perhaps that’s confusing and we should use (storage) space ? I just reached for memory space as its existing concept and I often used it to explain role account previously played

**Comment by gobengo** [comment]
> Perhaps that’s confusing and we should use (storage) space ?

Personally I think so. 

> Memory is what your computer uses to store data temporarily, while storage is where you save files permanently.
(via https://www.hp.com/us-en/shop/tech-takes/computer-memory-vs-storage )

In general it seems like we are trying to offer something closer to storage that is permanent/durable (as long as you pay your bill). To me in computers 'memory' usually implies RAM or some kind of temporary memory that may go away e.g. if the power goes out.
